src/net.h

```diff
@@ -590,6 +603,21 @@ class CNode
         return nRefCount;
     }
 
+    /** Returns true if a transaction with the passed properties will likely get accepted into this node's mempool */
+    bool IsTxAcceptable(const CTxProperties &props)
```

Comment:
is it "will likely get accepeted"  or is it really "will get accepted"?

Reply:
The transaction could be bad for reasons that are not checked by this function, the node could have lied about its nLimitAncestorCount or Size, could reject the tx based on fee, or basically for any unknown reason.